src/components/sound-editor/sound-editor.jsx

```diff
@@ -52,6 +54,16 @@ const messages = defineMessages({
         description: 'Title of the button to save trimmed sound',
         defaultMessage: 'Save'
     },
+    undo: {
```

Comment:
Kind of tangential, should we put strings like this in a shared location, so translators don't have to translate the string for undo in several places?